src/vaadin-message-input-text-area.d.ts

```diff
@@ -0,0 +1,21 @@
+import { TextAreaElement } from '@vaadin/vaadin-text-field/src/vaadin-text-area.js';
+
+/**
+ * The text area element for a message input.
+ *
+ * ### Styling
+ *
+ * See [`<vaadin-text-area>` documentation](https://github.com/vaadin/vaadin-text-field/blob/master/src/vaadin-text-area.js)
+ * for `<vaadin-message-input-text-area>` parts and available slots
+ *
+ * See [ThemableMixin – how to apply styles for shadow parts](https://github.com/vaadin/vaadin-themable-mixin#readme)
+ */
+declare class MessageInputTextAreaElement extends TextAreaElement {
+  _setAriaLabel(message: string): void;
```

Comment:
I would personally avoid adding this kind of internal function to our public API, although it's not private in the sense that it's used by another component in the module.

But not sure. I think it's the convention to have these things in the type defs.

Reply:
I also think we need to change this. Submitted https://github.com/vaadin/vaadin-messages/issues/54